apm-agent-plugins/apm-api-plugin/src/main/java/co/elastic/apm/plugin/api/TransactionInstrumentation.java

```diff
@@ -77,4 +78,23 @@ public static void setUser(@Advice.FieldValue(value = "span", typing = Assigner.
             transaction.setUser(id, email, username);
         }
     }
+
+    public static class MakeChildOfRumTransactionInstrumentation extends TransactionInstrumentation {
+        public MakeChildOfRumTransactionInstrumentation() {
```

Comment:
Stay consistent with naming based on the instrumented method name

Reply:
done